android/app/src/main/java/org/fossasia/openevent/api/processor/EventListResponseProcessor.java

```diff
@@ -26,82 +25,19 @@ private void save(Event event) {
     protected void onSuccess(Event event) {
         int counterRequests = 0;
 
-        final Version version = event.getVersion();
-
         final DataDownloadManager download = DataDownloadManager.getInstance();
 
-        Version storedVersion = realmRepo.getVersionIdsSync();
-
-        if (storedVersion == null) {
-            Timber.d("Version info not present. Downloading complete data again...");
-
-            counterRequests = 7;
-            OpenEventApp.postEventOnUIThread(new CounterEvent(counterRequests));
-
-            save(event);
-
-            download.downloadSession();
-            download.downloadSpeakers();
-            download.downloadTracks();
-            download.downloadMicrolocations();
-            download.downloadSponsors();
-            download.downloadSessionTypes();
-        } else {
-            if (storedVersion.getEventVer() != version.getEventVer()) {
-                Timber.d("Downloading Event");
-                save(event);
-
-                counterRequests++;
-            }
-
-            if (storedVersion.getSponsorsVer() != version.getSponsorsVer()) {
-                download.downloadSponsors();
-
-                Timber.d("Downloading Sponsor");
-                counterRequests++;
-            }
-
-            if (storedVersion.getSpeakersVer() != version.getSpeakersVer()) {
-                download.downloadSpeakers();
-
-                Timber.d("Downloading Speakers");
-                counterRequests++;
-            }
-
-            if (storedVersion.getSessionsVer() != version.getSessionsVer()) {
-                download.downloadSession();
-
-                Timber.d("Downloading Sessions");
-                counterRequests++;
-            }
-
-            if (storedVersion.getTracksVer() != version.getTracksVer()) {
-                download.downloadTracks();
-
-                Timber.d("Downloading Tracks");
-                counterRequests++;
-            }
-
-            if (storedVersion.getMicrolocationsVer() != version.getMicrolocationsVer()) {
-                download.downloadMicrolocations();
-                Timber.d("Downloading Microlocations");
-
-                counterRequests++;
-            }
-
-            if (storedVersion.getSessionTypesVer() != version.getSessionTypesVer()) {
-                download.downloadSessionTypes();
-                Timber.d("Downloading Session Types");
+	counterRequests = 7;
```

Comment:
@iamareebjamal i am outside currently.
@Shailesh351 please add a tab to this while fixing or otherwise i will be able to send a pr by night.